packages/assets-controllers/src/TokenRatesController.ts

```diff
@@ -221,7 +220,10 @@ export const getDefaultTokenRatesControllerState =
 
 /** The input to start polling for the {@link TokenRatesController} */
 export type TokenRatesPollingInput = {
-  chainId: Hex;
+  chainIdAndNativeCurrency: {
+    chainId: Hex;
+    nativeCurrency: string;
```

Comment:
Can the polling input still be only chainId? since we can get the `nativeCurrency` from `networkConfigurationsByChainId` ?

Reply:
Sorry could you clarify if the nativeCurrency we send from the UI will be retrieved from `networkConfigurationsByChainId`